CHANGELOG.md

```diff
@@ -95,6 +95,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 - Added `OnStride` special Lua function for `AHumans` that is called whenever they stride (i.e. when their `StrideSound` is played). Like playing `StrideSound`, this does not happen when the AHuman is climbing.
 
+- Added the helper function `RoundFloatToPrecision( )` to RTETools and exposed to Lua. Utility function to round and format floating point numbers for display in strings. 
```

Comment:
No need to mention adding to `RTETools`, and since it's exposed to Lua you should explain how to call it.

 
Added Lua utility function `RoundFloatToPrecision` to round and format floating point numbers for display in strings.
```
RoundFloatToPrecision(floatValue, digitsPastDecimal, roundingMode) -- Rounding mode 0 for system default, 1 for floored remainder, 2 for celied remainder
```
	.